Dockerfile

```diff
@@ -10,11 +10,15 @@ COPY analytics/src ./src
 COPY analytics/package.json .
 COPY analytics/pnpm-lock.yaml .
 COPY analytics/tsconfig.json .
+RUN mkdir /metrics
+VOLUME ["/metrics"]
+
 
 RUN pnpm install
 RUN pnpm run build
 
 RUN curl -vL --compressed https://github.com/XAMPPRocky/tokei/releases/download/v12.1.2/tokei-x86_64-unknown-linux-gnu.tar.gz --output tokei.tar.gz
 RUN tar -xvf tokei.tar.gz
+RUN ./tokei /repository --output json > /metrics/metrics.json
```

Comment:
For me, it's not clear how this run of tokei will result in a software project being analyzed.

Reply:
As we run a checkout action before running our action we will have copied the whole repository to be analysed into our own action Docker container under /repository at build time. we then also just compute metrics into a json at build time and use this file as an artifact to upload as a blob to the git refs and decode and use in the frontend.

This is arguably not the best solution as it does not allow for caching and reusing the image built for this action. Maybe we can get around this issue by mounting a volume instead in the future.